includes/Admin/Site_Health.php

```diff
@@ -230,11 +231,12 @@ public function add_extensions( array $core_extensions ): array {
 	 *
 	 * @since 1.8.0
 	 *
-	 * @param array $test_result Site Health test result.
+	 * @param array|mixed $test_result Site Health test result.
 	 *
-	 * @return array Modified test result.
+	 * @return array|mixed Modified test result.
 	 */
-	public function modify_test_result( array $test_result ): array {
+	public function modify_test_result( $test_result ) {
+		$test_result = (array) $test_result;
```

Comment:
The downside of blindly casting everything to an array is that if `$test_result` is `false` (if some plugin returned that for some reason), `$test_result` becomes

```
array(1) {
  [0]=>
  bool(false)
}
```

Same goes for other places with type casting.